airflow/utils/db.py

```diff
@@ -561,6 +562,37 @@ def initdb():
     Base.metadata.create_all(settings.engine)  # pylint: disable=no-member
 
 
+def wait_for_migrations(timeout):
+    """
+    Function to wait for all airflow migrations to complete. Used for launching airflow in k8s
```

Comment:
```suggestion
    Function to wait for all airflow migrations to complete.
```

It is nothing specific to k8s